packages/addons/browser/chrome/source/bin/chrome-downloader

```diff
@@ -53,13 +62,14 @@ rm -f ${CONTROL_FILE} ${DATA_FILE}
 ## extract chrome
 # extrat chrome.deb
 kodi-send --action="Notification(Extracting Chrome,starting,1000,${ICON})" >/dev/null
-ar -x google-chrome-stable_current_amd64.deb
+ar -x ${CHROME_FILE}
 
 # extract data.tar.xz to chrome-bin directory
 mkdir $ADDON_DIR/chrome-bin
 tar xf data.tar.xz --strip-components=4 -C $ADDON_DIR/chrome-bin ./opt/google/chrome
 
 # cleanup
+cd $ADDON_DIR
```

Comment:
what does this ?

Reply:
Leave `$ADDON_DIR/tmp_download` before removing.

Reply:
`rm -rf` should clean it regardless if I am at that folder or not !? 
btw the cd to the temp folder is questionable anyway, not sure why I did that (likely I forgot to rewrite it at some point), there is no need to cd to that folder at all ( i have a look about a fix)

Reply:
>rm -rf should clean it regardless if I am at that folder or not !?
Yes. It disappears from file system but the internal structures remain as long a reference is hold (by still having it as cwd). But no process can operate on this directory any more. I prefer changing the cwd to not having even to think about possible consequences.

The cd to the temp folder is there because of `ar -x ${CHROME_FILE}`.